templates/python/custom-copilot-rag-customize/src/requirements.txt

```diff
@@ -0,0 +1,5 @@
+python-dotenv
+aiohttp
+azure-search
+azure-search-documents
+teams-ai~=1.0.0
```

Comment:
may need to change teams-ai version to 1.0.1?

Reply:
updated